test/box-luatest/gh_3211_per_module_log_level_test.lua

```diff
@@ -42,12 +42,17 @@ g2.before_each(function(cg)
     end, {cg.params.cfg_type})
 end)
 
-g1.after_each(function(cg)
-    cg.server:drop()
-end)
-g2.after_each(function(cg)
+local function drop_server(cg)
+    cg.server:exec(function()
```

Comment:
You can use a special box.cfg wrapper for this task. https://www.tarantool.io/en/doc/latest/reference/reference_rock/luatest/classes/luatest.server/#server-update-box-cfg-cfg

Reply:
Done.